x/concentrated-liquidity/client/cli/tx.go

```diff
@@ -49,3 +50,13 @@ func NewWithdrawPositionCmd() (*osmocli.TxCliDesc, *types.MsgWithdrawPosition) {
 		Flags:               osmocli.FlagDesc{RequiredFlags: []*flag.FlagSet{FlagSetJustPoolId()}},
 	}, &types.MsgWithdrawPosition{}
 }
+
+func NewCollectFeesCmd() (*osmocli.TxCliDesc, *types.MsgCollectFees) {
+	return &osmocli.TxCliDesc{
+		Use:                 "collect-fees [lower-tick] [upper-tick]",
+		Short:               "collect fees from a liquidity position",
+		Example:             "collect-fees [-69082] 69082 --pool-id 1 --from val --chain-id osmosis-1",
```

Comment:
not sure if [] are necessary. I could be wrong
```suggestion
		Example:             "collect-fees -69082 69082 --pool-id 1 --from val --chain-id osmosis-1",
```

Reply:
It is for some dumb negative thing unfortunately.

Though I think having to list lower tick and upper tick is some not-great UX, and we will want to make a better CLI ux flow later

Reply:
Oh wow did not know that this was a thing that we need to do in order to use negatives in cli (and I think this is something important to have fixed, would folks mind if we create an issue for this to tackle whenever we have free capacity?)

Reply:
I actually had the same problem and created and [issue](https://github.com/osmosis-labs/osmosis/issues/4041)

On my side, using [negative value] does not work when I run cli commands, maybe it is because I am on osx, not sure. I proposed some solutions in the issue, would be great to have some feedback!